### ISSUE TITLE
Cannot find language server from atom flatpak install

### ISSUE THREAD

**Issue by ktaylora** [REditorSupport/atom-ide-r#10]
I'm using the flatpak installation of atom from flathub (https://flathub.org/apps/details/io.atom.Atom). I can install atom-ide-r just fine and have the languageserver 'R' package installed. But I can't get ide-r to find my R installation. This may be an issue with flatpak sandboxing, such that atom doesn't have permission to system paths? 

Anyhoo, atom-ide-r doesn't work out of the box for flatpak atom users.

**Comment by randy3k** [comment]
You could provide to the path to R in ide-r settings.

**Comment by ktaylora** [comment]
ide-r's default path to R is : /usr/bin/R . This path exists.

**Comment by randy3k** [comment]
did you also install atom-ide-ui?

**Comment by ktaylora** [comment]
Yes. atom-ide-ui is installed. I use it with ide-python on the same atom installation without a problem.

**Comment by randy3k** [comment]
The only advice I have for now is to check the R installation and confirm that languageserver has been installed.

**Comment by ktaylora** [comment]
That's fine. Thanks for considering. 

I'll keep this open until I figure out a fix or we get another Linux user that can offer-up a suggestion. And I'll reach out upstream to the atom flatpak team and see if I can get an answer out of them.
